CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Contributing Guidelines
+
+## Filing a Bug Report
+
+Simply create a new issue for a bug, and one of us at the VicRoy team will check it out!
+
+In your bug report please include the following 
+* A summary of what the bug is
+* The observed and expected behaviour
+* Steps to reproduce the bug
+* Other contextual information (e.g. version and operating environment)
+* Screenshots would also be great if possible
+
+## Suggesting a New Feature
```

Comment:
The issue templates are for bugs and feature requests, not PRs. May add a PR template later on, so I reckon for now its prob ok to just include basically the last sentence in the PR bit